oddbird/sass-site
# Playground initial, Indented formatting

## Description
Adds an initial Sass document, showing modules, lists, variables, and nesting. 

This also adds Formatting to the Indented mode, previously in #79 

## Related Issue(s)
#80

## Steps to test/reproduce
From homepage, click on Playground, and see there is content. Switch to Sass indented format, and see there is initial content in the indented format.

Make any changes, and reload, and see that your changes persist.

https://deploy-preview-81--sass-site-oddbird.netlify.app/playground

- [x] @jgerigmeyer  Review code
- [x] @mirisuzanne  Review default content- is it a good example?
